okhttp-testing-support/src/main/kotlin/okhttp3/ClientRuleEventListener.kt

```diff
@@ -266,7 +266,12 @@ class ClientRuleEventListener(
   }
 
   private fun logWithTime(message: String) {
-    val timeMs = TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - startNs)
+    val timeMs = if (startNs == 0L) {
```

Comment:
What’s going on here? Note that 0L is a valid timestamp!

Reply:
Early cancel.  I've updated and commented, preferring a masking variable that is immutable to startNs?.run or similar.